### PR TITLE
[IMP]sale_delivery_state: Skip sale order lines for delivery state

### PR DESCRIPTION
**Pull request by manuelregidor**
Based on https://github.com/OCA/sale-workflow/issues/2818

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Review by HaraldPanten — APPROVED** [review]
Functional review. Same as https://github.com/OCA/sale-workflow/pull/2845

@etobella

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

File: sale_delivery_state/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    skip_sale_delivery_state = fields.Boolean(
+        string="Skip Delivery State",
+        compute="_compute_skip_sale_delivery_state",
+        store=True,
```
Comment (simahawk) [inline sale_delivery_state/models/sale_order_line.py:13]: why stored?

Comment (etobella) [reply sale_delivery_state/models/sale_order_line.py:13]: this way you have the value on the moment you create the line. check that the compute does not depend on the company_id.skip_service_sale_delivery_state

**Pull request merged**